src/components/va-alert/va-alert.css

```diff
@@ -11,6 +11,33 @@
   background-color: unset;
 }
 
+:host([full-width='true']),
+:host([full-width='']) {
+  border-top-style: solid;
+  border-top-width: 10px;
+}
+
+:host([status='warning']) {
+  border-color: var(--color-gold);
+}
+:host([status='info']) {
+  border-color: var(--color-primary-alt-dark);
+}
+:host([status='error']) {
+  border-color: var(--color-secondary-dark);
+}
+:host([status='continue']),
+:host([status='success']) {
+  border-color: var(--color-green);
+}
```

Comment:
This benefits non `full-width` alerts as well. These replace the individual pieces of CSS down below:

```css
.[$STATUS] {
    border-color: var(--status-color);
}
```

This is possible due to the `border-color: inherit`.